src/actions/posts.ts

```diff
@@ -181,8 +181,13 @@ export function createPost(post: Post, files: any[] = []) {
             pending_post_id: pendingPostId,
             create_at: timestamp,
             update_at: timestamp,
+            reply_count: 0,
         };
 
+        if (post.root_id) {
+            newPost.reply_count = (Selectors.getPostRepliesCount(state, post.root_id) || 0) + 1;
```

Comment:
just a thought - maybe worth moving `|| 0` into the selector?

Reply:
good idea